R/expr-string.R

```diff
@@ -0,0 +1,1158 @@
+# The env for storing all expr str methods
+polars_expr_str_methods <- new.env(parent = emptyenv())
+
+namespace_expr_str <- function(x) {
+  self <- new.env(parent = emptyenv())
+  self$`_rexpr` <- x$`_rexpr`
+
+  lapply(names(polars_expr_str_methods), function(name) {
+    fn <- polars_expr_str_methods[[name]]
+    environment(fn) <- environment()
+    assign(name, fn, envir = self)
+  })
+
+  class(self) <- c(
+    "polars_namespace_expr", "polars_object"
+  )
+  self
+}
+
+#' Convert a String column into a Date/Datetime/Time column.
+#'
+#' Similar to the [strptime()] function.
+#'
+#' When parsing a Datetime the column precision will be inferred from the format
+#' string, if given, e.g.: `"%F %T%.3f"` => [`pl$Datetime("ms")`][pl_Datetime].
+#' If no fractional second component is found then the default is `"us"` (microsecond).
+#' @param dtype The data type to convert into. Can be either `pl$Date`,
+#' [`pl$Datetime()`][DataType_Datetime], or `pl$Time`.
```

Comment:
dtype page is not added yet.
Please add TODO like:
https://github.com/eitsupi/neo-r-polars/blob/800cdb866057b7f45292119b2b03ba99eed474c8/R/dataframe-frame.R#L1

Reply:
I'd rather not lose time on this since the link should be added later anyway and devtools will tell us if some links don't exist.

Reply:
Ah no my bad, `devtools` won't tell us